pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -263,6 +282,15 @@ func (r *ReconcileAPIManager) reconcileAPIManagerLogic(cr *appsv1alpha1.APIManag
 		}
 	}
 
+	reader := read.New(r.Client()).WithNamespace(cr.Namespace).WithOwnerObject(cr)
```

Comment:
What is the purpose of these few lines of code?

Reply:
Further investigated, they are redundant and removed.